src/ParaTest/Runners/PHPUnit/BaseRunner.php

```diff
@@ -1,7 +1,9 @@
 <?php
+
 namespace ParaTest\Runners\PHPUnit;
 
 use ParaTest\Coverage\CoverageMerger;
+use ParaTest\Coverage\CoverageReporter3;
```

Comment:
looks like this can safely be deleted, i don't see it anywhere in this file, PHPDoc or otherwise.